src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -153,6 +154,7 @@ internal static void Initialize(TaskLoggingHelper log)
                 ResolvedFrom = GetResourceFourSpaces("ResolveAssemblyReference.ResolvedFrom");
                 SearchedAssemblyFoldersEx = GetResourceEightSpaces("ResolveAssemblyReference.SearchedAssemblyFoldersEx");
                 SearchPath = EightSpaces + GetResource("ResolveAssemblyReference.SearchPath");
+                SearchPathAddedByParentAssembly = EightSpaces + GetResource("ResolveAssemblyReference.SearchPathAddedByParentAssembly");
```

Comment:
I don't know why there was one missed before, but might as well unify these

```suggestion
                SearchPath = GetResourceEightSpaces("ResolveAssemblyReference.SearchPath");
                SearchPathAddedByParentAssembly = GetResourceEightSpaces("ResolveAssemblyReference.SearchPathAddedByParentAssembly");
```